flink-ml-lib/src/main/java/org/apache/flink/ml/param/HasFeaturesCol.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.param;
```

Comment:
In Alink similar Params like `HasFeatureCol` is in package `org.apache.flink.ml.param.shared.colname`. Shall we also put this and other interfaces into corresponding packages?

Reply:
Discussed with @yunfengzhou-hub offline. We agreed there is no need to distinguish whether a param is a `colname` or not by having a dedicated `colname` subpackage. And we agreed to move those shared `WithParams` to the package `org.apache.flink.ml.common.param`. The `common` indicates `shared` just like Spark does.